core/remote/watcher/index.js

```diff
@@ -366,10 +371,9 @@ class RemoteWatcher {
   async applyAll(
     changes /*: Array<RemoteChange> */
   ) /*: Promise<Array<{ change: RemoteChange, err: Error }>> */ {
-    const errors = await Promise.mapSeries(
-      changes,
-      async change => await this.apply(change)
-    ).filter(err => err)
+    const errors = await Promise.mapSeries(changes, this.apply).filter(
```

Comment:
Don't we need to bind `this` on `this.apply`?

Reply:
It feels like we should indeed but no tests are failing :thinking:. I'll dig a bit.